keyboards/hotdox/keymaps/bliss/rules.mk

```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+EXTRAKEY_ENABLE = yes
+DYNAMIC_MACRO_ENABLE = yes
```

Comment:
I wouldn't recommend using this feature with via, as via has it's own macro support, too.